Utility.h

```diff
@@ -4,14 +4,17 @@
 #include <memory>
 #include "Item.h"
 
-
 std::vector<std::unique_ptr<Item>> makeHelpfulItems(int num);
 
 std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num);
 
+std::vector<std::unique_ptr<Item>> makeAttackItems(int num);
+
```

Comment:
The dragon slayer gets one attack item, not a vector of them. This function is unnecessary.
- [ ] remove